src/library-authoring/data/apiHooks.ts

```diff
@@ -101,6 +101,7 @@ export const useRevertLibraryChanges = () => {
     mutationFn: revertLibraryChanges,
     onSettled: (_data, _error, libraryId) => {
       queryClient.invalidateQueries({ queryKey: libraryAuthoringQueryKeys.contentLibrary(libraryId) });
+      queryClient.invalidateQueries({ queryKey: ['content_search'] });
```

Comment:
Q: is it really necessary to invalidate _all_ cached data related to content search?

Nit: it would be nice to convert this to the [query key pattern](https://github.com/openedx/frontend-app-course-authoring/blob/de3befec089bf9f9510601f74c66d29efc2e2dd4/src/taxonomy/data/apiHooks.js#L20-L23) like we're doing above for library authoring query keys.

Reply:
> Q: is it really necessary to invalidate all cached data related to content search?

@bradenmacdonald Actually, I only want to invalidate content search queries related to this library. So updated the new PR: https://github.com/openedx/frontend-app-course-authoring/pull/1214 to include this commit: https://github.com/openedx/frontend-app-course-authoring/pull/1214/commits/9a3fc044dec34dce13319f79acf6ed731a877a98 

I tried changing content search queries to use query key pattern but it has a lot of dynamic variables and I could not think of a nice way to implement it.